ui/app/AppLayouts/Communities/views/EditAirdropView.qml

```diff
@@ -512,27 +513,34 @@ StatusScrollView {
                 model: SortFilterProxyModel {
                     sourceModel: membersModel
 
+                    sorters : [
+                        StringSorter {
+                            roleName: "preferredDisplayName"
+                            caseSensitivity: Qt.CaseInsensitive
+                        }
+                    ]
+
                     filters: [
-                        ExpressionFilter {
+                        FastExpressionFilter {
                             enabled: membersDropdown.searchText !== ""
 
                             function matchesAlias(name, filter) {
                                 return name.split(" ").some(p => p.startsWith(filter))
                             }
```

Comment:
I wonder why alias needs to be handled in a special way here. If it's not necessary we could optimize it in a similar way as described here: https://github.com/status-im/status-desktop/pull/16363/files#r1774985033

Reply:
I tried this and it ended up being way slower actually :thinking: 
Maybe it's because there are so many properties?

Reply:
That's quite surprising :thinking: So let's leave it as it is :)

Bth, looking into code of `SFPM` I have a temptation to rewrite it from scratch :facepalm: